### PR TITLE
feat(rule): use valid aria rules

### PR DESCRIPTION
**Pull request by mohammedzamakhan**

### CODE REVIEW

File: test/templateAccessibilityValidAriaRule.spec.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { getFailureMessage, Rule } from '../src/templateAccessibilityValidAriaRule';
+import { assertAnnotated, assertSuccess } from './testHelper';
+
+const {
+  metadata: { ruleName }
+} = Rule;
+
+describe(ruleName, () => {
+  describe('failure', () => {
+    it('should fail when aria attributes are misspelt or does not exists', () => {
```
Comment (mgechev) [inline test/templateAccessibilityValidAriaRule.spec.ts]: misspelt -> misspelled

Comment (wKoza) [reply test/templateAccessibilityValidAriaRule.spec.ts]: exists -> exist

File: src/util/getSuggestion.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import * as editDistance from 'damerau-levenshtein';
+
+const THRESHOLD = 2;
+
+export const getSuggestion = (word: string, dictionary: string[] = [], limit = 2) => {
```
Comment (wKoza) [inline src/util/getSuggestion.ts]: Can you add a test of this ?

File: src/index.ts [hunk 3]
```diff
@@ -28,8 +28,12 @@ export { Rule as PreferInlineDecorator } from './preferInlineDecoratorRule';
 export { Rule as PreferOutputReadonlyRule } from './preferOutputReadonlyRule';
 export { Rule as TemplateConditionalComplexityRule } from './templateConditionalComplexityRule';
 export { Rule as TemplateCyclomaticComplexityRule } from './templateCyclomaticComplexityRule';
+<<<<<<< HEAD
 export { Rule as TemplateAccessibilityTabindexNoPositiveRule } from './templateAccessibilityTabindexNoPositiveRule';
 export { Rule as TemplateAccessibilityLabelForVisitor } from './templateAccessibilityLabelForRule';
+=======
```
Comment (mgechev) [inline src/index.ts]: @mohammedzamakhan there are some merge conflict markers here

**Pull request merged**